### PR TITLE
Make prizes optional.

### PR DESCRIPTION
**Pull request by Pike**
Prizes are nice, but not every event can really afford to hand them out.
Let's make them optional.

It'd be great if we could make use of this for https://mozweekend.de/ on the 12th, and for the buglist preps leading up to that.

cc @MichaelKohler

### CODE REVIEW

File: reclama/templates/bugs.html [hunk 1]
```diff
@@ -51,11 +51,11 @@
             <a href="{{ b.bug_url }}" target="_blank" class="bug-title" id="b{{ b.number }}"></a>
           </h3>
           <div id="m{{ b.number }}" class="bug-modified"></div>
-        </div>
+        </div> {% if b.prize %}
         <div class="bug-prize"
```
Comment (comzeradd) [inline reclama/templates/bugs.html]: Please use a new line here to create the `if` block and indent accordingly.

File: reclama/templates/bugs.html [hunk 2]
```diff
@@ -51,11 +51,11 @@
...
              title="If you have fixed this bug before the end of the event, please come by the Mozilla booth to pick up your prize.">
           Prize: {{ b.prize }}
-        </div>
+        </div> {% endif %}
```
Comment (comzeradd) [inline reclama/templates/bugs.html]: Same here.

**Comment by comzeradd** [comment]
Yes, it makes sense to have this as optional.

@MichaelKohler ping [Infrastructure team](https://wiki.mozilla.org/Participation/Infrastructure) if you want to run a bugs sprint, to provide necessary access.

**Comment by Pike** [comment]
Follow up commit pushed to address the styling comments.

**Comment by comzeradd** [comment]
Could you please squash the two commits to one?

**Comment by Pike** [comment]
Squashed.

**Comment by comzeradd** [comment]
Thanks for your contribution. Commit is merged.

**Comment by brianking** [comment]
@comzeradd will this be live for this weekend?

**Comment by comzeradd** [comment]
@brianking yeap. we'll merge a couple of more fixes and we'll deploy on time.

**Pull request merged**